specs/VersionInfo/VersionInfo.md

```diff
@@ -0,0 +1,188 @@
+
+```c
+#include <WindowsAppSDK-VersionInfo.h>
+#include <MddBootstrap.h>
+// ...
+
+HRESULT hr = MddBootstrapInitialize(WINDOWSAPPSDK_RELEASE_MAJORMINOR, WINDOWSAPPSDK_RELEASE_VERSION_TAG_W, WINDOWSAPPSDK_RUNTIME_VERSION_UINT64);
+if (FAILED(hr))
+{
+    wprintf(L"Error 0x%X in Bootstrap initialization\n", hr);
+}
+```
+
+where `WINDOWSAPPSDK_RELEASE_MAJORMINOR`, `WINDOWSAPPSDK_RELEASE_VERSION_TAG_W` and
+`WINDOWSAPPSDK_RUNTIME_VERSION_UINT64` are build-time constants from `WindowsAppSDK-VersionInfo.h`
+i.e.
+
+```c
+MddBootstrapInitialize(WINDOWSAPPSDK_RELEASE_MAJORMINOR, WINDOWSAPPSDK_RELEASE_VERSION_TAG_W, WINDOWSAPPSDK_RUNTIME_VERSION_UINT64)
+```
+
+for 1.1.0 is equivalent to
+
+```c
+MddBootstrapInitialize(0x00010001, L"", 0x03E80204086C0000u)
+```
+
+Developers using the Bootstrapper auto-initializer link a global constructor into their executable
+which calls the Bootstrapper API with this build-time information.
+
+All these options share a common property: the developer specifies the WinAppSDK release, channel
+and a _minimum_ acceptable package version. This resolves to a specific package at runtime that
+satisfies the dependency which is >= MinVersion (not ==MinVersion).
+
+One can indirectly determine the actual runtime version in use by retrieving the package graph via
+[GetCurrentPackageInfo()](https://docs.microsoft.com/windows/win32/api/appmodel/nf-appmodel-getcurrentpackageinfo)
+and walking the result looking for a framework package in the runtime's family. This is cumbersome
+(even more so for C# given the need for P/Invoke and the shape of the API for C# callers), can pose
+performance overhead (searching across a deep copy of the package graph) and is potentially
+ambiguous (e.g. advanced scenarios using Dynamic Dependencies can have multiple runtimes in the
+package graph).
+
+This can be shrinkwrapped via an easy to use API where we (WinAppSDK) incur the burden of
+implementation complexity, using a design to directly and efficiently yield the correct answer.
+
+This provides a simple but reliable and performant solution for applications, as well as class
+libraries and other components using WinAppSDK.
+
+# Examples
+
+An example using the runtime version information API to display the actual version in use at runtime:
+
+```c#
+using System;
+using System.Windows.Forms;
+using Microsoft.Windows.ApplicationModel.WindowsAppRuntime;
+
+public class MyInfo
+{
+    public static void About()
+    {
+        var build_release = Microsoft.WindowsAppSDK.Release;
+        var build_version = Microsoft.WindowsAppSDK.Runtime.Version;
+        var release = VersionInfo.ReleaseInfo.Current();
+        var runtime = .VersionInfo.RuntimeInfo.Current();
```

Comment:
>.Ve [](http://example.com/codeflow?start=22&length=3)

It looks like there is an extra "." ".VersionInfo.RuntimeInfo.Current()" should be "VersionInfo.RuntimeInfo.Current()"?

Reply:
RECOMMEND: Remove the extra dot